src/services/registration.js

```diff
@@ -0,0 +1,32 @@
+// const regModel = require('../models/registrationExample');
+const checkName = require('../services/checkUserName');
+const checkPsw = require('../services/checkPassword');
+const dbCheck = require('../services/regCheckDB');
+const users = require('../models/userDB');
+
+const createUser = item => new Promise((resolve) => {
+  const temp = {
+    username: '',
+    password: '',
+  };
+  if (checkName.CheckUserName(item.username) && checkPsw.CheckPassword(item.password)) {
+    if (item.password === item.confirmPsw) {
+      if (!dbCheck.containsUser(item, users.userDB)) {
```

Comment:
This is badly structured. If you truly had a DB, you wouldn't ask for the entire DB content (possibly millions of data) to check if your item is in it. You need to refactor this service call.
First name your dbCheck to dbService.
Then refactor the containsUser to a function that expects a single parameter and rewrite the service function so that it can work like this.

Reply:
I refactored it. I hope it is okay now :D